bootcamp/tutorials/quickstart/quickstart_with_attu.md

```diff
@@ -21,16 +21,24 @@ sudo xattr -rd com.apple.quarantine /Applications/attu.app
 ```
 
 ---
-
-## 3. Connect to Zilliz Cloud
+## 3. Connect to Milvus
+Attu supports connecting to both **Milvus Standalone** and **Zilliz Cloud**, providing flexibility to work with local or cloud-hosted databases.
```

Comment:
Shall be self hosted Milvus instead.

Attu supports both Milvus Standalone and Milvus Distributed.